### PR TITLE
Add chainId, networkId to rest function call data.

### PR DESCRIPTION
**Pull request by liayoo**
Note that before using the chainId, a function server should verify using ain-util's ecVerifySig() that the transaction body, signature, address, and chainId are valid.

### CODE REVIEW

**Review by minsulee2 — APPROVED** [review]
LGTM!

**Comment by liayoo** [comment]
Thanks for the fast review! Now merging~

**Comment by minsulee2** [comment]
Any issues with unittest btw?

File: db/functions.js [hunk 1]
```diff
@@ -255,6 +255,8 @@ class Functions {
               blockNumber,
               blockTime,
               auth: newAuth,
+              chainId: blockchainParams.chainId,
```
Comment (platfowner) [inline db/functions.js:258]: 👍

**Pull request merged**